src/web/core/src/main/java/org/geoserver/web/data/layergroup/LayerGroupEditPage.java

```diff
@@ -34,38 +72,222 @@ public LayerGroupEditPage(PageParameters parameters) {
             return;
         }
         
-        initUI(lg);
-
-        if (!isAuthenticatedAsAdmin()) {
-            Form f = (Form)get("form");
+        setupPublished(lg);
+        postInit();
+    }
     
-            //global layer groups only editable by full admin
-            if (lg.getWorkspace() == null) {
-                //disable all form components but cancel
-                f.visitChildren(new IVisitor<Component>() {
-                    @Override
-                    public Object component(Component c) {
-                        if (!(c instanceof AbstractLink && "cancel".equals(c.getId()))) {
-                            c.setEnabled(false);
+    private void postInit() {
+        if (!isAuthenticatedAsAdmin() && !isNew) {
+            // global layer groups only editable by full admin
+            if (getPublishedInfo().getWorkspace() == null) {
+                // disable all form components but cancel
+                setInputEnabled(false);
+
+                info(new StringResourceModel("globalLayerGroupReadOnly", this, null).getString());
+            }
+        }
+    }
+            
+    public class LayerGroupTab extends PublishedEditTabPanel<LayerGroupInfo> {
+
+        private static final long serialVersionUID = 2192005814142588155L;
+
+        public LayerGroupTab(String id) {
+            super(id, myModel);
+            initUI();
+        }
+
+        private EnvelopePanel envelopePanel;       
+        protected RootLayerEntryPanel rootLayerPanel;    
+        
+        private void initUI() {
+
+            final WebMarkupContainer rootLayerPanelContainer = new WebMarkupContainer("rootLayerContainer");
+            rootLayerPanelContainer.setOutputMarkupId(true);
+            add(rootLayerPanelContainer);        
+            
+            rootLayerPanel = new RootLayerEntryPanel("rootLayer", getPublishedInfo().getWorkspace(), myModel);
+            rootLayerPanelContainer.add(rootLayerPanel);
+
+            updateRootLayerPanel(getPublishedInfo().getMode());
+            
+            TextField<String> name = new TextField<String>("name");
+            name.setRequired(true);
+            //JD: don't need this, this is validated at the catalog level
+            //name.add(new GroupNameValidator());
```

Comment:
While it's true that it's validated at the catalog level,  the GroupNameValidator provides internationalized error messages, the catalog exceptions do not... @jdeolive, do you remember if there was any other reason to comment this out?